### ISSUE TITLE
Logarithmic colorscales in maps

### ISSUE THREAD

**Issue by HansKallekleiv** [equinor/webviz-subsurface-components#1996]
Do we currently support logarithmic color sampling for maps in the Subsurface Viewer?

**Comment by hkfb** [comment]
Yes, you can choose logarithmic color sampling in [this example](https://equinor.github.io/webviz-subsurface-components/storybook-static/?path=/story/subsurfaceviewer-map-layer-colormap--color-selector)

**Comment by hkfb** [comment]
That's probably right, it would have to be added. Either by supporting callbacks (I remember we discussed this some time ago. Is it possible with Dash?) or adding some prop to control color sampling.
